test/helpers/misc.rb

```diff
@@ -118,3 +118,9 @@ def wait_until_not_nil(give_up_after = 3, &block)
     current_tries += 1
   end
 end
+
+def skip_unless_minitest5_or_above
```

Comment:
The method name has me thinking Minitest5 wouldn't be skipped, but with `>` version 5, would Minitest 5 also be skipped?

Reply:
+1

Reply:
Ah, I can see that. We're doing an alphabetical sort comparison here (as we do elsewhere in the agent).

The constant will hold a value like `"5.16.3"` or even `"5.0.0"` and when we do a string comparison with `>` to `"5"`, all v5 versions will satisfy the check.

I'll give some thought to a helper method in future that makes our agent's use of string comparisons for something that seems numeric to be consistent and clearer.

Reply:
Ooh right, I forgot string comparison feels unintuitive sometimes lol. Thanks for explaining!

Reply:
Ooo right right you've mentioned this before. Thank you!